docs/builders.rst

```diff
@@ -163,3 +163,53 @@ or
 .. hint::
 
     As an alternative, you can set the config option :ref:`needs_build_needumls` to export the needumls files during each build.
+
+.. _needs_lut_builder:
+
+needs_lut
+---------
+.. versionadded:: 1.4.0
+
+The **needs_lut** builder exports all found needs to a single json file, which only include list of key ``id`` and value of ``docname`` or ``external_url``.
```

Comment:
We should mention what `lut` stands for. Maybe:

``**needs_lut** (Needs Lookup Table``)